Apollo.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:22-alpine3.20
```

Comment:
Why apollo dockerfile uses nodejs v22 and dockerfile uses the nodejs v23 ?
All dockerfiles in the project should have the same nodejs version